html-aam/index.html

```diff
@@ -3016,8 +3016,14 @@ <h4 id="el-img">`img`</h4>
             <tr>
               <th>[[wai-aria-1.2]]</th>
               <td>
-                <a class="core-mapping" href="#role-map-image">`image`</a>
-                or <a class="core-mapping" href="#role-map-img">`img`</a> role
+                <div>
+                  If element has a non-empty `src` or `srcset` attribute, an <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a>, or both, then map to the
+                  <a class="core-mapping" href="#role-map-image">`image`</a> or <a class="core-mapping" href="#role-map-img">`img`</a> role.
```

Comment:
```suggestion
                  <a class="core-mapping" href="#role-map-image">`image`</a> or <a class="core-mapping" href="#role-map-img">`image`</a> role.
```

Reply:
refer to the `image` role vs the original `img` role, as image has become the primary name for these synonyms
https://w3c.github.io/aria/#image